src/components/TrendChartDetails.js

```diff
@@ -16,16 +16,10 @@ const getComparison = ({ place, data }) => {
   const placeRate = data[place].rate
   const nationalRate = data[nationalKey].rate
   const diff = (placeRate / nationalRate - 1) * 100
-  const diffAbs = Math.abs(diff)
 
-  return diffAbs < threshold
-    ? <span>{highlight(`about the same (within ${threshold}%)`)} as</span>
-    : <span>
-        {highlight(
-          `${formatRate(diffAbs)}% ${diff > threshold ? 'higher' : 'lower'}`,
-        )}
-        {' '}than
-      </span>
+  return Math.abs(diff) < threshold
+    ? <span>the same as</span>
+    : <span>{highlight(`${diff > 0 ? 'higher' : 'lower'}`)} than</span>
```

Comment:
I think the copy here should specify the threshold that we're using to determine if it is "higher", "lower", or "about the same". Without that information, it seems like we're simply editorializing.

Before this PR, the copy was something like "The rate for that year was about the same (within 3%) that of the United States." Now it reads "The rate for that year was about the same that of the United States".

What do you think, @RyanSibley @brendansudol @amberwreed?